### ISSUE TITLE
aria-modal="true" is misspelled attribute?

### ISSUE THREAD

**Issue by kalpeshsingh** [GoogleChrome/lighthouse#3282]
Lighthouse shows "**[aria-*] attributes are valid and not misspelled.**" message under "ARIA Attributes Follow Best Practices" when you try to use `aria-modal="true"`.

`aria-modal="true"` attribute is recommended by W3C (WAI-ARIA) for modals. You can read more details on WAI-ARIA [Authoring Practices 1.1](https://www.w3.org/TR/wai-aria-practices/#dialog_modal). 

Lighthouse should not mark this attribute as wrong or misspelled attribute. It is very much needed attribute for modals. 

<img width="923" alt="screen shot 2017-09-09 at 1 48 18 am" src="https://user-images.githubusercontent.com/13119410/30230230-fbec9722-9502-11e7-8b08-b5c7627601bc.png">

**Comment by ebidel** [comment]
Can you provide the url you tested?

cc @robdodson

**Comment by paulirish** [comment]
Hmm good call...

I can repro this on this minimal page: http://output.jsbin.com/toxugemahi/1/quiet

The spec seems to indicate `true` is valid. http://w3c.github.io/aria/aria/aria.html#aria-modal

It appears axe's `lib/commons/aria/index.js` is missing details on aria-modal..

**Comment by paulirish** [comment]
aria-modal was just added to axe here: https://github.com/dequelabs/axe-core/commit/ebb2a5d545fae5f309dbc1efa2780e89d29e6597

It's in v2.4.0 and v3.0alpha. 

@robdodson will be bumping our version soon so we'll get this sorted.

**Comment by paulirish** [comment]
and the fast fix!  :)

**Comment by kalpeshsingh** [comment]
Haha. This is super fast fix. 
Thank you @paulirish @ebidel and @robdodson 
You guys are awesome!
